volatility3/framework/layers/intel.py

```diff
@@ -102,6 +102,10 @@ def _translate(self, offset: int) -> Tuple[int, int, str]:
         translated address lives in and the layer_name that the address
         lives in
         """
+        if not self.is_address_canonical(offset):
```

Comment:
This will have no effect, since the all places the private method `_translate` are called are now preceded by a call to `get_canonical_address`.

Reply:
Removed this check